diagnostics/forcing_feedback/doc/forcing_feedback.rst

```diff
@@ -0,0 +1,90 @@
+
+Contact info
+------------
+
+- PI of the project: Brian Soden, University of Miami (bsoden@rsmas.miami.edu);
+- Current developer: Ryan Kramer (ryan.j.kramer@nasa.gov)
+
+Open source copyright agreement
+^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^^
+
+This package is distributed under the LGPLv3 license (see LICENSE.txt).
+
+Functionality
+-------------
+
+The currently package consists of:
+- a Python script (forcing_feedback.py), which sets up the directories and calls\.\.\.
+- \.\.\. an Python script (forcing_feedback_kernelcalcs.py) which reads the data, performs the calculations and saves the results to temporary netcdfs./././.
+- \.\.\. Finally, a Python script (forcing_feedback_plots.py) reads in the temporary results, observational radiative forcing and feedbacks and creates plots.  Throughout the package, the scripts use Python functions defined in a third Python script (forcing_feedback_util.py)
+
+As a module of the MDTF code package, all scripts of this package can be found
+under ``mdtf/MDTF_$ver/var_code/forcing_feedback``
+and pre-digested observational data and radiative kernels (in netcdf format) under ``mdtf/inputdata/obs_data/forcing_feedback``
+Place your input data at: ``mdtf/inputdata/model/$model_name/mon/``
+
+Required programming language and libraries
+-------------------------------------------
+
+Python is required to run the diagnostic.
+
+The part of the package written in Python requires packages os, sys, numpy, xarray, scipy, matplotlib and cartopy. These Python packages are already included in the standard Anaconda installation 
+
+Required model output variables
+-------------------------------
+
+The following three 3-D (lat-lon-time), monthly model fields are required:
+- surface temperature ("ts" in CMIP conventions)
```

Comment:
Do you mean "tas", which is the 2-m air temperature, or "ts" here which is skin temp?

Reply:
We are using "ts" skin temp but were recently talking about switching to "tas". We've found it really doesn't make a noticeable difference in terms of the feedback calculations, however.  Do you have a preference for which we use?  At least in CMIP-land it does seem like tas is a more common variable.